### DISCUSSION TITLE
Loop through a single album per day, but cycle through a list of albums daily

### DISCUSSION THREAD

**Discussion by lennon101** [damongolding/immich-kiosk#87] [Feature request]
**Is your feature request related to a problem? Please describe.**
No. 

**Describe the solution you'd like**
I can see in the FAQ that you have listed how to setup multiple albums. But my guess is that this simple means that the immich-kiosk will cycle through all the photos in that album and then move to the next album and cycle through all of those photos and so on and so forth? What I would like is to loop through a single album on day 1, then on day 2 begin looping through another album, and so on until there are now more albums and we loop back to the first album on day 1. 

**Describe alternatives you've considered**
No alternatives at this stage. 

**Additional context**
I think the functionality is mostly already there, but there just needs to be some ability to control when the next album is selected

**Comment by damongolding** [comment]
At the moment because users can set album(s) and people at the same time, Kiosk adds all the requested album(s) ID(s) and people IDs into a "bucket". Then Kiosk grabs one at random and fetches an image related to that album or person.
